src/drivers/tone_alarm/CMakeLists.txt

```diff
@@ -34,8 +34,11 @@
 px4_add_module(
 	MODULE drivers__tone_alarm
 	MAIN tone_alarm
+	COMPILE_FLAGS
+		#-DDEBUG_BUILD
 	SRCS
 		ToneAlarm.cpp
+		ToneAlarm.h
```

Comment:
@dagar , I can build successfully without this line and bench tests show no difference.

Reply:
It only exists to help IDEs with cmake integration.